content/deploy/community-cloud/status-and-limitations.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Status and limitations
+slug: /deploy/streamlit-community-cloud/status
+---
+
+# Status and limitations of Community Cloud
+
+## Community Cloud Status
+
+You can view the current status of Community Cloud at [streamlitstatus.com](https://www.streamlitstatus.com/).
+
+## GitHub OAuth scope
+
+To deploy your app, Streamlit requires access to your app's source code in GitHub and the ability to manage the public keys associated with your repositories. The default GitHub OAuth scopes are sufficient to work with apps in public GitHub repositories. However, to access your private repositories, we create a read-only [GitHub Deploy Key](https://docs.github.com/en/free-pro-team@latest/developers/overview/managing-deploy-keys#deploy-keys) and then access your repo using a public SSH key. When we create this key, GitHub notifies repo admins of the creation as a security measure.
+
+Streamlit requires the additional `repo` OAuth scope from GitHub to work with your private repos and manage deploy keys. We recognize that the `repo` scope provides Streamlit with extra permissions that we do not really need and which, as people who prize security, we'd rather not even be granted. Alas, we need to work with the APIs we are provided by GitHub.
```

Comment:
I would probably take out the "Alas..." part. GitHub has been providing a better API (since about 2 years ago), but we have not prioritized addressing this.